src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void setDone() {
+        this.isDone = true;
+    }
```

Comment:
Good job following naming conventions, however, I think you may have to place a parameter for the boolean instead.